Library/Formula/libutf8proc.rb

```diff
@@ -0,0 +1,25 @@
+class Libutf8proc < Formula
```

Comment:
Sorry, can you leave this name as `utf8proc` as that's the name of the project and we don't mandate prefixing `lib` unless the upstream project does. You could always create an alias to `libutf8proc` if you like.